dockers/docker-fpm-frr/frr/bgpd/bgpd.main.conf.j2

```diff
@@ -8,13 +8,21 @@
 ! TSA configuration
 !
 ip prefix-list PL_LoopbackV4 permit {{ get_ipv4_loopback_address(LOOPBACK_INTERFACE, "Loopback0") | ip }}/32
+{% if multi_npu()  == true %}
+  ip prefix-list PL_LoopbackV4 permit {{ get_ipv4_loopback_address(LOOPBACK_INTERFACE, "Loopback4096") | ip }}/32
+{% endif %}
```

Comment:
why do you need it here?

Reply:
@pavel-shirshov, I think this might be needed to support TSA for the multi-asic platforms. 
Let me know if my understanding is wrong. I will remove this change

Reply:
@arlakshm I think you're not going to announce your internal loopbacks at all, right? This prefix-list is used to announce loopback only from the device in TSA mode.

Reply:
Removed in the latest commit